docs/development/build-linux.md

```diff
@@ -73,3 +73,8 @@ The following additional packages are installed:
 * pkg-config
 * libgtk2.0-dev
 * zip
+
+
+Use 
+sudo apt-get install build-essential automake libtool git curl flex bison libx11-dev libxext-dev libxrender-dev libxft-dev libxinerama-dev libxv-dev libxcursor-dev libfreetype6-dev libpopt-dev libesd0-dev liblcms-dev pkg-config libgtk2.0-dev zip
```

Comment:
I suggest using a markdown code block here. e.g.
```
  sudo apt-get install ...
```

Also it would probably be best separated into several lines for easy copy/pasting into a terminal.